libs/dal/src/repositories/topic/topic-subscribers.repository.ts

```diff
@@ -16,4 +16,22 @@ export class TopicSubscribersRepository extends BaseRepository<EnforceEnvironmen
   constructor() {
     super(TopicSubscribers, TopicSubscribersEntity);
   }
+
+  async addSubscribers(entity: TopicSubscribersEntity): Promise<void> {
+    const { _environmentId, _organizationId, _topicId, _userId, subscribers } = entity;
+
+    await this.update(
+      {
+        _environmentId,
+        _organizationId,
+        _topicId,
+        _userId,
+      },
+      {
+        $addToSet: { subscribers },
```

Comment:
This ensure the uniqueness of every entry for subscribers so we avoid to have repeated subscribers in the topic, therefore avoid notification duplication to the subscriber.